conf/ds-present-behaviour.yaml

```diff
@@ -90,19 +91,20 @@ storages:
             root: 'src/test/resources/xml/corpus' # Expected
             # Whether or not remove ID-prefix (the part from the beginning to the first `:` (inclusive) of the ID)
             # from the ID when perforaing the lookup. Optional, default is true.
-            stripprefix: true
+            prefix:
+              strip: true
         - folder:
             root: '../src/test/resources/xml/corpus' # Maybe PWD is target/
         - folder:
             root: 'ds-present/src/test/resources/xml/corpus' # Maybe PWD is outsid eof the checkout (unlikely)
 
 # Settings for handling export of multiple records
 records:
-  # If true, export is stopped if a transformation fails for any record
-  # If false, failed records are logged and skipped
-  # Default: true
-  stoponerrors: true
-
+  errors:
+    # If true, export is stopped if a transformation fails for any record
+    # If false, failed records are logged and skipped
+    # Default: true
+    stop: true
```

Comment:
I like that this is more YAML'esque, however, i feel like it is hard to decode the meaning of the path `errors.stop` - would a primary key like errorHandling etc. make more sense?